examples/rsp-cra-18/package.json

```diff
@@ -3,15 +3,18 @@
   "version": "0.1.0",
   "private": true,
   "homepage": ".",
+  "packageManager": "yarn@4.2.2",
   "dependencies": {
     "@adobe/react-spectrum": "latest",
     "@babel/plugin-proposal-private-property-in-object": "^7.16.7",
     "@babel/preset-react": "^7.24.1",
-    "@react-aria/dnd": "^3.5.0",
+    "@react-aria/dnd": "^3.6.1",
     "@react-spectrum/color": "^3.0.0-beta.28",
+    "@react-spectrum/inlinealert": "^3.0.0-rc.0",
+    "@react-spectrum/provider": "^3.9.7",
```

Comment:
where did these come from? Aren't they part of the mono package?

Reply:
provider is a peer to everything, therefore we have to include it
inlinealert should be removed, we aren't using it directly